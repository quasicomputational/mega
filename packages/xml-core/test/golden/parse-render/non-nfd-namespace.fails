<foo xmlns="example:Ã©">hello</foo>