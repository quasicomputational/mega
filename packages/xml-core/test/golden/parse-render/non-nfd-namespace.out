Warning: Non-ASCII literal characters at (1:21-1:22).
Error: Namespace is not in NFD at (1:6-1:11).
